### ISSUE TITLE
oc binary included within 4.16.0-rc4 not working on RHEL 8.10

### ISSUE THREAD

**Issue by adrianriobo** [crc-org/snc#910]
It seems the oc binary included within bundle 4.16.0-rc4 has been built on a RHEL 9. As so when trying to run it from a RHEL 8.X (8.10 in this case) we are hitting  GLIBC versions missmatch

```bash
 oc: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by oc) oc: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by oc) oc: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by oc) 
```

**Comment by praveenkumar** [comment]
Looks like https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.16.0-rc.4/ now have dedicated binary for RHEL-8 and default is switched to RHEL-9

**Comment by adrianriobo** [comment]
Fixed by #911
